src/sage/crypto/public_key/key_exchange/anshel_anshel_goldfeld.py

```diff
@@ -0,0 +1,50 @@
+import random
+from sage.crypto.public_key.key_exchange import KeyExchange
+
+class AnshelAnshelGoldfeld(KeyExchange):
```

Comment:
Can you split this file into another branch? I want to merge the rest of this before I'll have time to read up on this scheme so I can review it.